numpy/linalg/linalg.py

```diff
@@ -1041,7 +1041,7 @@ def eigvals(a):
     >>> LA.norm(Q[0, :]), LA.norm(Q[1, :]), np.dot(Q[0, :],Q[1, :])
     (1.0, 1.0, 0.0)
 
-    Now multiply a diagonal matrix by Q on one side and by Q.T on the other:
+    Now multiply a diagonal matrix by `Q` on one side and by `Q.T` on the other:
```

Comment:
Double backticks maybe needed here. @mattip?

Reply:
`Q` is not a parameter, it is a new variable. So this is a very short code block, and double backticks would be better.

Reply:
Thank you! I will update the PR.

Reply:
Is there anywhere else in the PR I should have double backticks instead of single? It's been a while since I used Sphinx and I don't have great intuition for when to use each.

Reply:
Double back-ticks render as code. Single back-ticks are shortcuts for references depending on context. If this is a docstring and the content is a parameter, single back-ticks mean "parameter". Otherwise sphinx will try to find a reference with the content and link to it.